tests/test_hello_world.py

```diff
@@ -0,0 +1,14 @@
+from application import create_app
+import pytest
+import json
+
+def test_get_main_namespace():
+    flask_app = create_app()
+    flask_app.testing = True
+    
+    with flask_app.test_client() as test_client:
```

Comment:
@Edison-Stuart You'll need this test client all over the place once you start writing the rest of this service. You should make this a fixture in `confest.py`. Something like:

```
@pytest.fixture()
def mock_app_client():
    app = create_app()
    mongoengine.connect("somealias", mongo_client_class=mongomock.MongoClient)
    yield app.test_client()
    mongoengine.disconnect("somealias")
```

so that your test becomes something like

```
def test_get_main_namespace(mock_app_client):
    json_response = mock_app_client.get(
            "/api/v1/main/hello"
    )
    assert json_response.status_code == 200
```